Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -218,12 +211,14 @@ class TrackStateProxy {
   /// @note The mask parameter will not cause a copy of components that are
   ///       not allocated in the source track state proxy.
   template <bool RO = ReadOnly, bool ReadOnlyOther,
-            typename = std::enable_if<!RO>>
-  void copyFrom(const TrackStateProxy<M, ReadOnlyOther>& other,
+            typename = std::enable_if_t<!RO>>
+  void copyFrom(const TrackStateProxy<Trajectory, M, ReadOnlyOther>& other,
                 TrackStatePropMask mask = TrackStatePropMask::All,
                 bool onlyAllocated = true) {
     using PM = TrackStatePropMask;
 
+    // @TODO: How to support arbitrary columns here?
```

Comment:
todo?

Reply:
Haven't figured it out yet. Would leave this on in for now.